connectors/src/connectors/google_drive/index.ts

```diff
@@ -825,3 +825,12 @@ export async function googleDriveGarbageCollect(connectorId: ModelId) {
 
   return launchGoogleGarbageCollector(connectorId);
 }
+
+export async function pauseGoogleDriveWebhooks(connectorId: ModelId) {
```

Comment:
Any reason to have Webhooks in the name of the function? I'd be in favor of using generic names, like for other connectors.